udisks/mount_options.conf.in

```diff
@@ -0,0 +1,25 @@
+# This file contains custom mount options for udisks 2.x
```

Comment:
The commit message says this file is optional, so what's our plan with this for Fedora/RHEL? Are we going to install it to /et/udisks2? I think it could be useful to have it there.

Reply:
Generally it's up to distributors how they handle the file. Such config files that are shipped only as an example may be placed under `/usr/share/doc/udisks-2.9.0/` or as /etc/udisks2/mount_options.conf.example`.

I haven't found a specific guideline in https://docs.fedoraproject.org/en-US/packaging-guidelines/ but if you grep `/usr/share/doc` for `example` on Fedora, you may see a couple of packages doing that. On the contrary there are none in `/etc` on my system.

My point is that when a config file is not required for regular operation, it should rather be renamed to avoid unnecessary parsing all the time.

TODO: modify `Makefile.am` and install the file somewhere perhaps...

Reply:
I've few `.example` and `.sample` configs in my `/etc` so it should be ok in Fedora. I also prefer this way -- nobody is going to look to `/usr/share/doc`, our config could be confusing and having sample config in `/etc` would definitely be useful when creating a custom one.

Reply:
Alright, added install hooks to move the file at `/etc/udisks2/mount_options.conf.example`. The file will still appear as `mount_options.conf` in the source tree so that we can run some tests over it.